views/js/lib/mathquill/mathquill.css

```diff
@@ -170,6 +170,65 @@
   border-top: 1px solid black;
   margin-top: 1px;
 }
+.mq-math-mode .mq-overleftrightarrow {
+    margin-top: 4px;
+    position: relative;
+    border-top: 1px black solid;
+}
```

Comment:
Should we move our own CSS modifications somewhere else? Same reason as the other file.